automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Table.scala

```diff
@@ -67,4 +71,19 @@ case class Table(private val id: String)(implicit webDriver: WebDriver)
     val rows = tableBody.webElement.findElements(By.cssSelector("[data-test-class='table-row']")).toList
     rows.map(_.findElements(By.cssSelector("[data-test-class='table-cell']")).toList.map(_.getText))
   }
+
+  def readColumnHeaders: List[String] = {
+    awaitReady()
+    readAllText(columnHeaders)
+  }
+
+  def hideColumn(header: String) = {
+    if (readAllText(columnHeaders).contains(header)) {
+      columnEditorButton.doClick
+      val colToBeHidden = Checkbox(s"$header-column-toggle")
+      colToBeHidden.ensureUnchecked()
+      val action = new Actions(webDriver)
+      action.sendKeys(Keys.ESCAPE)
```

Comment:
This needs to be `action.sendKeys(Keys.ESCAPE).perform()`, otherwise the keypress is only queued, not actually performed. (There's also a `build()` function that returns a `CompositeAction` that you can then call `perform()` on repeatedly.)